modules/formulize/class/durationElement.php

```diff
@@ -0,0 +1,271 @@
+<?php
+require_once XOOPS_ROOT_PATH . "/modules/formulize/class/elements.php";
+require_once XOOPS_ROOT_PATH . "/modules/formulize/include/functions.php";
+
+class formulizeDurationElement extends formulizeElement
+{
+
+	function __construct()
+	{
+		$this->name = "Duration";
+		$this->hasData = true;
+		$this->needsDataType = false; // We'll force this to be stored as INT (minutes)
+		$this->overrideDataType = "int"; // Store as integer minutes
+		$this->adminCanMakeRequired = true;
+		$this->alwaysValidateInputs = true; // We'll always validate to ensure duration is within bounds
+		$this->canHaveMultipleValues = false;
+		$this->hasMultipleOptions = false;
+		parent::__construct();
+	}
+}
+
+class formulizeDurationElementHandler extends formulizeElementsHandler
+{
+
+	var $db;
+
+	function __construct($db)
+	{
+		$this->db = &$db;
+	}
+
+	// Conversion factors to minutes
```

Comment:
Perhaps a style thing, but wouldn't it be more typical to have the properties defined prior to the constructor? I don't care a lot as long as this works.